core/src/smartcontracts/isi/world.rs

```diff
@@ -201,6 +201,10 @@ pub mod isi {
         }
     }
 
+    // TODO: Looks like it no longer makes sense to keep this:
+    // 1. We have a permission token for every operation in validator;
+    // 2. If someone registers a new token at runtime, then it has no way to
+    // specify how to validate its granting or revoking.
     impl Execute for Register<PermissionTokenDefinition> {
```

Comment:
It's a hard question. I think it will be useful for runtime migrations later. I think, I'll delete this comment, because restriction from validator side will be enough

Reply:
I'd reference this in a new issue.

Reply:
Created #3381